functionary/ui/templates/tags/output_table.html

```diff
@@ -0,0 +1,20 @@
+{% if format == "table" %}
+    <div class="table-container">
+        <table id="result_table" class="table is-hoverable is-striped">
+            <thead>
+                <tr>
+                    {% for header in result.headers %}<th>{{ header }}</th>{% endfor %}
+                </tr>
+            </thead>
+            <tbody>
+                {% for row in result.data %}
+                    <tr>
+                        {% for col in row %}<td>{{ col }}</td>{% endfor %}
+                    </tr>
+                {% endfor %}
+            </tbody>
+        </table>
+    </div>
+{% else %}
+    <pre>{{ result | linebreaksbr }}</pre>
```

Comment:
Seeing this in both templates strikes me as weird.  Should this just be its own `output_raw.html` template that gets rendered if the output_format is neither of json or html?

Reply:
That was an artifact from not having separate output templates. It's not accessible now so it has been removed.